pkgs/tools/misc/pkgdiff/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, perl, wdiff }:
+stdenv.mkDerivation rec{
+  version = "1.7.2";
+  name = "pkdiff-${version}";
+
+  src = fetchFromGitHub {
+    owner = "lvc";
+    repo = "pkgdiff";
+    rev = version;
+    sha256 = "1ahknyx0s54frbd3gqh070lkv3j1b344jrs6m6p1s1lgwbd70vnb";
+  };
+
+  buildInputs = [ perl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  postInstall = ''
```

Comment:
```suggestion
  postInstall = ''
    patchShebangs $out/bin/pkgdiff";
```
to avoid
```
[nix-shell:/home/jon/.cache/nix-review/pr-72312]$ ./result/bin/pkgdiff --help
./result/bin/pkgdiff: /nix/store/xwbvgxp05yh9agb1598l25yqwj12440i-pkdiff-1.7.2/bin/.pkgdiff-wrapped: /usr/bin/perl: bad interpreter: No such file or directory
```

Reply:
@jonringer 
Can you explain, why this is necessary?
My .pkdiff-wrapped already has the patched interpreter line, without calling patchShebangs explicitly:
```
╭─ simon@simon-desktop  ~                                                                                                    ✔  10194  10:10:53
╰─ head /nix/store/gahyv2fkly0m3ckdqyspfg8s01wy7xk3-pkdiff-1.7.2/bin/.pkgdiff-wrapped
#!/nix/store/5r85g4zzk0djffzs47zaimvkxfmbpp3q-perl-5.30.0/bin/perl
###########################################################################
# PkgDiff - Package Changes Analyzer 1.7.2
# A tool for visualizing changes in Linux software packages
#
# Copyright (C) 2012-2016 Andrey Ponomarenko's ABI Laboratory
#
# Written by Andrey Ponomarenko
#
# PLATFORMS
```